Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1817,9 +1817,14 @@
 
 (define (kernel-mig-routines-blocked-in-lockdown-mode-avoid-telemetry) (kernel-mig-routine
     thread_info
+#if HAVE(HARDENED_MACH_EXCEPTIONS)
+    task_register_hardened_exception_handler
+    thread_adopt_exception_handler
```

Comment:
I think we need to check if these are defined:

    (when (defined? 'task_register_hardened_exception_handler) task_register_hardened_exception_handler)
    (when (defined? 'thread_adopt_exception_handler) thread_adopt_exception_handler)